ci-droid-tasks-consumer-infrastructure/src/main/java/com/societegenerale/cidroid/tasks/consumer/infrastructure/config/CiDroidBehavior.java

```diff
@@ -6,12 +6,17 @@
 
 import java.util.Map;
 
+import static org.apache.commons.lang3.StringUtils.EMPTY;
+
 @Component
 @Data
 @ConfigurationProperties(prefix = "cidroid-behavior")
 public class CiDroidBehavior {
 
     private Map<String, String> patternToResourceMapping;
 
+    private int maxFilesInPr = Integer.MAX_VALUE;
```

Comment:
Let's use sensible default values, so that it can be used out of the box, and tweaked if required - can we have the values you put in properties copied there as default ?

Reply:
ok. My idea was not to introduce a new feature on release.
Updated the values